h/zos.h

```diff
@@ -1123,6 +1123,44 @@ typedef struct TIOTEntry_tag{
   int       tioejfcb;      /* this evil thing is 24 high bits of SWA index, and last byte of status bits */
 } TIOTEntry;
 
+/* IDTA Version */
```

Comment:
Can you point to the source this comes from?

Reply:
Done